topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -0,0 +1,629 @@
+>    - In *"5' (Front) Adapters"*:
+>        - Click on *"Insert 5' (Front) Adapters"*:
+>        - In *"1: 5' (Front) Adapters"*:
+>            - *"Source"*: `Enter custom sequence`
+>                - *"Enter custom 5' adapter sequence"*: `CTTCCGATCTACAAGTT`
+>        - Click on *"Insert 5' (Front) Adapters"*:
+>        - In *"2: 5' (Front) Adapters"*:
+>            - *"Source"*: `Enter custom sequence`
+>                - *"Enter custom 5' adapter sequence"*: `CTTCCGATCTTGGTCCT`
+>    - *"Minimum overlap length"*: `5`
+>    - *"Output filtering options"*: `Set Filters`
+>        - *"Minimum length"*: `10`
+>    - *"Additional output options"*: `Default`
+>    - *"Additional modifications to reads"*: `No Read Modifications`
+>
+>    > ### {% icon comment %} Do the same thing for the input control data set.
+>    >
+>    > If you processed the RBFOX2 fastq dataset then do the same thing for the input control dataset or *vice verca*.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Removal of UMIs with **UMI-tools extract**
+
+In this task we are going to remove the UMI at the 5' end of the second read.
+
+> ### {% icon hands_on %} Hands-on: UMI Removal
+>
+> 1. **UMI-tools extract** {% icon tool %} with the following parameters:
+>    - *"Library type"*: `Paired-end`
+>     - {% icon param-file %} *"Reads in FASTQ format"*: `fastq R2` (output from the **second** **Cutadapt** call {% icon tool %})
+>     - {% icon param-file %} *"Reads in FASTQ format"*: `fastq R1`  (output from the **second** **Cutadapt** call {% icon tool %})
+>     - *"Barcode on both reads?"*: `Barcode on first read only`
+>    - *"Use Known Barcodes?"*: `No`
+>    - *"Method to extract barcodes"*: `String`
+>    - *"Barcode pattern for first read"*: `NNNNN`
+>    - *"Is the barcode at the 5' end?"*: `Yes`
+>    - *"Output log?"*: `Yes`
+>    - *"Enable quality filter?"*: `No`
+>
+>    > ### {% icon comment %} Do the same thing for the input control data set.
+>    >
+>    > If you processed the RBFOX2 fastq dataset then do the same thing for the input control dataset or *vice verca*.
+>    {: .comment}
+>
+{: .hands_on}
+
+# Step 4: Aligning Reads to a Reference Genome
+
+To determine where DNA fragments originated in the genome, the sequenced reads must be aligned to a reference genome. This is equivalent to solving a jigsaw puzzle, but unfortunately, not all pieces are unique. In principle, you could do a BLAST analysis to figure out where the sequenced pieces fit best in the known genome. However, this can take a couple of weeks. Nowadays, there are many read alignment programs. `STAR` is one of them that works well with CLIP-Seq data, for more information read  [here](https://doi.org/10.1093/bioinformatics/bts635). `STAR` is able to use genome as well as transcriptome data. This ability is handy, since CLIP-Seq generates transcriptome data, consequently we have to take RNA processing steps like splicing events into account.
```

Comment:
`for more information read` ... I'm not sure this should refer to a paper. I would expect other tutorials or readthedocs kind of thing. If you want to link a paper, could we say something like `for more information see the original publication` or just simple link the publication. Some above for a nature paper.